test/to_vec.jl

```diff
@@ -117,12 +118,27 @@ end
         end
 
         @testset "Factorizations" begin
-            for dims in [(5, 5), (4, 6), (7, 3)]
+            # (100, 100) is needed to test for the NaNs that can appear in the
+            # qr(M).T matrix
+            for dims in [(5, 5), (4, 6), (7, 3), (100, 100)]
```

Comment:
I don't think it is necessary to test for that many different sizes. Just one size greater than 36x36 should be enough.

Reply:
My idea was to test one all combinations of square-rectangular matrices for `Q` and `R`, just to be on the safe side.

Reply:
Maybe just:
```suggestion
            for dims in [(7, 3), (100, 100)]
```
? Those other two shouldn't hit any different paths, right? My only concern is that since we already test for different combinations of `T` here, testing lots of other combinations can quickly blow up the time these tests take with little improvement to actual coverage, if they all hit the same paths anyways. Not a huge deal right now, but as the test suite grows, that can become an issue, so I'd prefer to be more conservative here.